xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -59,10 +59,23 @@ void CGUIWindowPVRGuide::OnInitWindow()
 
   CGUIEPGGridContainer *epgGridContainer =
     dynamic_cast<CGUIEPGGridContainer*>(GetControl(m_viewControl.GetCurrentControl()));
+
   if (epgGridContainer)
-    epgGridContainer->GoToNow();
+  {
+    CPVRChannelGroupPtr group(GetGroup());
+    if (!group)
+      group = g_PVRManager.GetPlayingGroup(m_bRadio);
```

Comment:
`GetGroup()` already returns a valid group so there is no need for this check here.

see https://github.com/xbmc/xbmc/blob/master/xbmc/pvr/windows/GUIWindowPVRBase.cpp#L166